barcodes_search/wizards/barcode_search.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017-Today GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class BarcodeSearch(models.TransientModel):
+    _name = 'barcode.search'
+
+    barcode = fields.Char(
+        string='Barcode', required=True, help='Barcode to search for.')
```

Comment:
Not a big deal, but field declarations are much easier to read and maintain if they're defined one per line.